manufacturing/enclosed_build/display_panel/README.md

```diff
@@ -1,18 +1,39 @@
-# Display panel
+# Display Panel
 
-![](images/rendering.jpg)
+| Assembly CAD Rendering | Assembled Back View |
+|:-------------------------:|:-------------------------:|
+| ![](images/display_panel_assembly_rendering.jpg) | ![](images/through-fitting_2.jpg) |
 
-## Upper Face Plate
+|Download Type|Link|
+|:------|:-------|
+|Display Panel Assembly CAD|[.iam](display_panel_assembly.iam)|
+|Display Assembly CAD|[.iam](display_assembly.iam)|
+|#TODO - Display Panel Assembly Drawing PDF|
 
-**#TODO: explain design rationale and link to CAD files here. Make it as good or better as the other pages!**
+## Overview
 
-The sizing for the mounting holes was based on [this](https://www.pemnet.com/fastening_products/pdf/sidata.pdf) catalog
-for heat set insert.
-M4 screws were chosen to mount the acrylic face plate to the sheet metal enclosure.
-For corresponding M4 nuts (refer page 6, metric table, row 6), length code 1 was chosen since acrylic thickness
-(6.35 mm) was more than minimum hole depth specification (5.72 mm).
-Hence the corresponding outer diameter for insertion i.e. acrylic mounting diameter would be 5.94 mm, which is
-reflected in the CAD.
+This readme contains the details, BOM, assembly instructions for one display panel assembly for the enclosed build. 
+
+## Design Rationale
+
+The display panel accommodates the LCD touchscreen display and the oxygen inlet port. The display panel assembly should provide an easily accessible way to the primary user interface - the touchscreen display. It accomplishes this by mounting the display assembly in a custom cut transparent acrylic sheet which also allows one to view the internals.  
+### Display Panel mounting choice 
+It was decided that heat set inserts be used to help screw the display panel to the sheet metal enclosure tray. This was done since the holes in the acrylic lacked any internal threads. Tapped/plastic threads are easy to overload and strip out. Metal threads provide a reliable way of fastening plastic and acrylic parts. The heat set inserts will resist pull-out and torque-out loads due to knurled external surface features designed to interface with the bulk acrylic. Also, using screws and nuts wasn't very feasible due to the lack of space between the mounted display (i.e. when attached to the sheet metal equipment tray) and the front face of the sheet metal enclosure. The sizing for the mounting holes was based on [this](https://www.pemnet.com/fastening_products/pdf/sidata.pdf) catalog for heat set insert. M4 screws were chosen to mount the acrylic face plate to the sheet metal enclosure. For corresponding M4 nuts (refer page 6, metric table, row 6), length code 1 was chosen since acrylic thickness (6.35 mm) was more than minimum hole depth specification (5.72 mm). Hence the corresponding outer diameter for insertion i.e. acrylic mounting diameter would be 5.94 mm, which is reflected in the CAD.
```

Comment:
This paragraph is too much. Maybe make 2 of it.

Also, divide your lines rather than use wrap. See how the other files are. One long line makes it impossible to comment on specific thing in review.

"it was decided" :/
Start with the fundamental assumptions and then work down into the decision of choice of part.

When you link something, identify it "PEM corporation parts catalog" or whatever it is called not "this".